shell/common/persistent_cache_unittests.cc

```diff
@@ -124,6 +129,8 @@ TEST_F(ShellTest, CacheSkSLWorks) {
   };
   fml::VisitFiles(dir.fd(), remove_visitor);
   DestroyShell(std::move(shell));
+
+#endif // defined(SHELL_ENABLE_GL)
```

Comment:
The `#endif` should be at least before `// Remove all files generated`. Otherwise, there might be some warnings about temp files not removed during the test.

BTW, would the test pass we if only put `#ifdef` around the single line of `ASSERT_EQ(skp_count, old_skp_count);`?

Reply:
Yeah I think it probably will, I'll check today

Reply:
Sorry for the delay, got distracted. I wasn't able to wrap that one line because of an unused variable warning on old_skp_count, butI tightened up this ifdef as much as I could without otherwise refactoring the test.